cmd/ctl/ctl_test.go

```diff
@@ -103,3 +103,51 @@ func (suite *CtlTestSuite) TestPruneChecksResults() {
 	suite.Equal(1, len(prunedChecksResults))
 	suite.Equal(int64(3), prunedChecksResults[0].ID)
 }
+
+func (suite *CtlTestSuite) TestGetLatestEvents() {
+	suite.tx.AutoMigrate(&datapipeline.DataCollectedEvent{})
+
+	events := []datapipeline.DataCollectedEvent{
+		{
+			ID:            1,
+			AgentID:       "agent_id_1",
+			DiscoveryType: "discovery_type_1",
+			Payload:       []byte("{}"),
+		},
+		{
+			ID:            2,
+			AgentID:       "agent_id_1",
+			DiscoveryType: "discovery_type_1",
+			Payload:       []byte("{}"),
+		},
+
+		{
+			ID:            3,
+			AgentID:       "agent_id_2",
+			DiscoveryType: "discovery_type_2",
+			Payload:       []byte("{}"),
+		},
+		{
+			ID:            4,
+			AgentID:       "agent_id_2",
+			DiscoveryType: "discovery_type_2",
+			Payload:       []byte("{}"),
+		},
+		{
+			ID:            5,
+			AgentID:       "agent_id_2",
+			DiscoveryType: "discovery_type_2",
+			Payload:       []byte("{}"),
+		},
```

Comment:
Can we add an additional discovery here in the test with a different discovery type?
To test if the latest event with other discovery type is retrieved as well.
Simply add a new entry with `discovery_type_3` for any of this 2 agents, and check if it is obtained

Reply:
@arbulu89 test added